qa-covid19.planx-pla.net/manifest.json

```diff
@@ -6,26 +6,28 @@
   "versions": {
     "arborist": "quay.io/cdis/arborist:2020.03",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
+    "covid19-etl": "quay.io/cdis/covid19-etl:1.0.0",
```

Comment:
that's not the same version as prod, but it's ok. we are releasing a new version tomorrow anyway